docs/source/api/service.md

```diff
@@ -10,5 +10,5 @@
 
 ```{eval-rst}
 .. autoconfigurable:: Service
-   :members: name, admin, url, api_token, managed, kind, command, cwd, environment, user, oauth_client_id, server, prefix, proxy_spec
+   :members:
```

Comment:
Previews:

- This PR: https://jupyterhub--4253.org.readthedocs.build/en/4253/api/service.html#service
- Main branch: https://jupyterhub.readthedocs.io/en/latest/api/service.html#service